templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Manufacturer</h1>
+
+  <p>Are you sure you want to delete the car: {{ manufacturer }}</p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger" />
+  </form>
```

Comment:
add a cancel button as well. it should redirect to the previous page the user was on